test/infrastructure/docker/internal/provisioning/ignition/kindadapter.go

```diff
@@ -88,7 +88,7 @@ func getActions(userData []byte) ([]provisioning.Cmd, error) {
 
 	for _, u := range ignition.Systemd.Units {
 		contents := strings.TrimSpace(u.Contents)
-		path := filepath.Join("/etc/systemd/system", u.Name)
+		path := fmt.Sprintf("/etc/systemd/system/%s", u.Name)
```

Comment:
Why is this change necessary? Shouldn't `filepath.Join` be used when dealing with joining paths?

Reply:
I think filepath.Join doesn't really help as the separator is already hard-coded in the first path (but not sure if that's the idea behind the finding)

Reply:
Yep, along with what @sbueringer says, it doesn't make sense in this context because we never need to account for Windows path separators. There will never be a case where we are running on an OS where the path would need to be `\etc\systemd\system` rather than `/etc/systemd/system`.

Reply:
Fair :+1: